bauhaus/workflows/secondary/hqMetrics.py

```diff
@@ -16,66 +17,135 @@ def conditionTableType():
 
     def generate(self, pflow, ct):
         outputDict = {}
-        pflow.bundleScript("R/hqrf_pbbamr.R")
+        rscript = "R/hqrf_pbbamr.R"
+        pflow.bundleScript(rscript)
+        controls = "fasta/600bp_Control_c2.fasta"
+        pflow.bundleScript(controls)
```

Comment:
I would like to avoid bundling it, but it's normally found in /etc/pacbio and installed with RPMS on PAC machines.  I don't know the reason for that.